blackbox/requirements.txt

```
@@ -1,7 +1,7 @@
 # don't pin crate version numbers so the latest will always be pulled when you
 # set up your environment from scratch
 
-crash<0.25.0
+crash
```

Can also nail it like this, because it apparently works on this release?
```suggestion
crash==0.30.0
```

---

with crash==0.30.0 we still get the same message about enterprise version

---

Yeah that's true, that won't go away by switching crash, because it is embedded into the test case here. It is just that `crash<0.25.0` _needed_ to be relaxed, and I was contemplating about whether relaxing/pinning it explicitly to `crash==0.30.0` would be better than without.